www/include/reporting/dashboard/xmlInformations/common-Func.php

```diff
@@ -92,7 +92,7 @@ function fillBuffer($statesTab, $row, $color)
     $detailPopup = '{table class=bulleDashtab}';
     $detailPopup .= '{tr}{td class=bulleDashleft colspan=3}' . $Day .
         ': {span class ="isTimestamp isDate"}';
-    $detailPopup .= $date_start . '{/span} --  ' . $Duration . ': '.
+    $detailPopup .= date('Y-m-d', $date_start) . '{/span} --  ' . $Duration . ': '.
```

Comment:
These labels classes : "isTimestamp" & "isDate" were used to manage the localization and was working since the 18.10.
IMHO, you should check why moment is unable to convert the timestamp